Sources/Afterpay/Views/Alerts.swift

```diff
@@ -15,18 +15,28 @@ enum Alerts {
     retry: @escaping () -> Void,
     cancel: @escaping () -> Void
   ) -> UIAlertController {
+    let localisedMessage = Afterpay.string.localised.alert.failedToLoadMessage
+
     let alert = UIAlertController(
-      title: "Error",
-      message: "Failed to load Afterpay checkout",
+      title: Afterpay.string.localised.alert.errorTitle,
+      message: String.localizedStringWithFormat(localisedMessage, "Afterpay"),
```

Comment:
Example of one reason to prefer US spelling - here we have both `localized` and `localised` on the same line.

Reply:
Addressed in [8f196d0](https://github.com/afterpay/sdk-ios/pull/207/commits/8f196d03f2504520b355b0714154a32340141aae)